plugins/wallet_plugin/wallet_plugin.cpp

```diff
@@ -54,7 +54,6 @@ void wallet_plugin::plugin_initialize(const variables_map& options) {
    }
    if (options.count("eosio-key")) {
       std::string eosio_wif_key = options.at("eosio-key").as<std::string>();
-      eosio_wif_key = fc::json::from_string(eosio_wif_key).as<std::string>();
```

Comment:
I could never figure out why this line was required. I had no problems setting a key once it was removed. It was a little annoying when present though because the empty key would always have to be specified with quotes. For example `eosio-key = ""` or `--eosio-key '""'`. Once removing this line I could simply leave `eosio-key = ` or `--eosio-key ""`. Setting the key still seemed to work fine.